File: client-development-app/src/Classes/Grocery.ts
export default class Grocery {

    name : string;
    type : string;

    constructor(name : string, type : string){
        this.name = name;
        this.type = type;
    }

    getName() {
        return this.name;
    }

    getType() {
        return this.type;
    }
}

File: GroceryCart/mobile-app/src/GroceryCart/app/src/main/java/com/example/grocerycart/Grocery.java
package com.example.grocerycart;

public class Grocery {

    String name;
    String type;

    public Grocery(String name, String type){
        this.name = name;
        this.type = type;
    }

    public String getName(){
        return this.name;
    }

    public String getType(){
        return this.type;
    }

}
